src/handleAction.js

```diff
@@ -7,19 +7,14 @@ export default function handleAction(type, reducers, defaultState) {
     ? type.toString()
     : type;
 
+  const nextReducer = isFunction(reducers) ? reducers : reducers.next;
+  const throwReducer = isFunction(reducers) ? reducers : reducers.throw;
```

Comment:
What do you think about using multiple assignment to DRY things up here?

In the [Babel REPL with our `.babelrc`](https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-0&code=let%20foo%0Alet%20bar%0Aconst%20o%20%3D%20%7B%20foo%3A%20'baz'%2C%20bar%3A%20'qux'%20%7D%0A%0Aif%20(false)%20%7B%0A%20%20%5B%20foo%2C%20bar%20%5D%20%3D%20%5B%20o.foo%2C%20o.bar%20%5D%20%0A%7D%20else%20%7B%0A%20%20%5B%20foo%2C%20bar%20%5D%20%3D%20%5B%20o%2C%20o%20%5D%20%0A%7D%0A%0A%0Aconsole.log(foo%2C%20bar)):

```js
let foo
let bar
const o = { foo: 'baz', bar: 'qux' }

if (flag) {
  [ foo, bar ] = [ o.foo, o.bar ] 
} else {
  [ foo, bar ] = [ o, o ] 
}

// flag === true => baz qux
// flag === false =>  {"foo":"baz","bar":"qux"} {"foo":"baz","bar":"qux"}
console.log(foo, bar) 
```

so we could be more DRY here with

```js
let nextReducer = throwReducer = reducers;

if (!isFunction(reducers)) {
  [ nextReducer, throwReducer ] = [ reducers.next, reducers.throw ]
}
```